### DISCUSSION TITLE
Wireless action camera(DJI or Gopro) as video source and mic attached to the phone as audio source - is it possible?

### DISCUSSION THREAD

**Discussion by kungaa** [datarhei/restreamer#539] [General]
So latest action cameras support sending RTMP stream over wifi. Attaching the mic to the action camera would be an easy solution. But the thing is - I want to use DJI Action 2's camera module only, without an additional part to which you can connect the mic. 

So I am thinking - what if I attach a lav mic to my phone, somehow stream audio from the lav mic, and mix it inside Restreamer? If it is possible - what app would you suggest to stream audio from a phone to Restreamer? Will be a desync between video and audio a huge problem? I hoped there is some sort of RTMP/SRT relay app for phones, where you can mix different audio and video sources locally before sending it to Restreamer, but I am guessing such an app does not exist. 

The closest thing I have seen to my idea was Belabox with Jetson Nano, where you can point the RTMP stream from the action camera to Belabox and attach a wireless lav mic to Jetson Nano and feed the audio stream to Belabox too. But that setup is too bulky and power-hungry.

**Comment by svenerbeck** [comment]
Hi kungaa,

Your setup sounds interesting, and you already have some excellent ideas! As you mentioned, using a Belabox with Jetson Nano may be a practical solution. However, the setup may be too bulky and energy-intensive for your needs.

An alternative option could be to use a restreamer with a battery pack. Then, if you choose SRT (Secure Reliable Transport), you can transfer latency-free to another restreamer or OBS. This should help you optimize your setup while getting the performance you want.

In terms of using a phone to stream audio from a Lavalier mic, there may be apps that can help you with this. For example, you can look for apps that support RTMP or SRT streaming and can send audio from your phone to the restreamer. Larix Broadcaster is an example of such an app, but other alternatives exist.

Regarding the synchronization of video and audio streams, there might be latency differences between the two sources. The extent of this problem depends on various factors, including the quality of the Internet connection and the hardware used. However, you may be able to manually adjust the synchronization in Restreamer or OBS to compensate for any delays.

Good luck with your project; hopefully, you will find the right solution for your needs!

Cheerio
Sven
